src/less/mixins/private/dropdown-mixins.less

```diff
@@ -9,13 +9,29 @@
     .box-shadow-token(~"@{component}-box-shadow", dropdown-box-shadow);
     box-sizing: border-box;
     display: none;
+    left: 0;
     max-height: 400px;
     min-width: 100%;
     overflow-y: auto;
     position: absolute;
     top: calc(100% + 4px);
     width: auto;
     z-index: 2;
+
+    [dir="rtl"] & {
+        left: unset;
+        right: 0;
+    }
+}
+
+.dropdown-reverse(@component) {
```

Comment:
Hmm...is this arg doing anything? Doesn't look necessary.

Reply:
Agreed, this argument isn't needed. 
The rest of the PR looks good though.